Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+#!groovy
```

Comment:
@seandst - Are Jenkinsfiles supposed to be in this repo?

Reply:
[Noooooooooooooooooope](https://www.youtube.com/watch?v=mJXYMDu6dpY)

Reply:
The Jenkinsfile came from meta_skeleton. I'll go ahead and remove it though

Reply:
Huh? What? What Jenkinsfile? There's nothing to see here.

*sweeps meta_skeleton git history under the rug*